DFReader.py

```diff
@@ -689,17 +692,13 @@ def _parse_next(self):
         profiler.add_function(DFReader_binary._add_msg)
         profiler.add_function(DFReader._set_time)
         profiler.enable_by_count()
-                    
+
     filename = sys.argv[1]
-    if filename.endswith('.log'):
```

Comment:
This change makes things less clear.

Further, considering I'm vaguely working on a DFReader_CSV, this would need to be reverted later.